atlp-rwanda/e-commerce-furebo-32-bn
# Reset Password via email

#### What does this PR do?

Reset Password via email

#### Describe your changes

This PR adds the functionality to reset a user's password via email. When a user forgets their password, they can request a reset, and an email with a reset link will be sent to them. This link allows the user to reset their password securely.

#### PR Type
What kind of change does this PR introduce?
```
[ ] Bugfix
[x] Feature
[ ] Refactoring (no functional changes, no API changes)
[ ] Documentation content changes
[ ] Tests
[ ] Other


#### How should this be manually tested?

Go to the login page and click on "Forgot Password".
Enter the registered email address.
Check the email inbox for a reset link.
Click on the reset link and set a new password.
Verify that the new password works for logging into the application.

#### What are the relevant pivotal tracker stories?
[Pivotal Tracker Link](https://xsolve.atlassian.net/browse/PROJECT_KEY)

#### Screenshots (if is appropriate)
[ ] Feature
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Tests
[ ] Other
```

#### How should this be manually tested?

Navigate to the login page and click on "Forgot Password".
Enter the registered email address.
Check the email inbox for a reset link.
Click on the reset link and set a new password.
Verify that the new password works for logging into the application.


#### What are the relevant pivotal tracker stories?
[Pivotal Tracker Link](https://xsolve.atlassian.net/browse/PROJECT_KEY)

#### Screenshots (if is approp
![ffw](https://github.com/atlp-rwanda/e-commerce-furebo-32-bn/assets/111005167/d4004842-e98d-4c6b-b7fb-551e276b3696)
riate)
